README.md

```diff
@@ -1,68 +1,58 @@
+* Install [Visual Studio](https://visualstudio.microsoft.com/) with "Desktop development with C++" to include MSVC toolchain
 
-## dev dependencies
-* [service-fabric-cpp](https://github.com/youyuanwu/service-fabric-cpp) (fetched automatically by cmake) and its dependencies
+## Getting Started - Ubuntu
+<em>Note: Service Fabric currently only supports Ubuntu 18.04 LTS and Ubuntu 20.04 LTS.</em>
 
-## Quick Start/Build
-Build all rust lib and examples.
-```
-cmake . -B build
-cmake --build build
+* Install Service Fabric Runtime for Linux (Ubuntu). See [Prepare your development environment on Linux](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started-linux?tabs=sdksetupubuntu%2Clocalclusteroneboxcontainer)
+* Following the documentation, proceed to **Manual installation** and skip **Step 6. Add Azul JDK Key**..
+
+## Getting Started - Ubuntu on WSL
+The setup is similar to the regular Ubuntu setup, but with some twicks to avoid Service Fabric installer to search for Windows mount paths, which can slow down installation.
+
+Remove `/mnt/c` paths to speed up installation by adding the following to `/etc/wsl.conf` and restart WSL to apply the changes:
+```sh
+# This removes windows path
+[interop]
+appendWindowsPath = false
+# This do not mount windows drive
+[automount]
+enabled = false
 ```
-* For linux, vscode devcontainer is maintained for development.
 
-## Build details
-### Setup prerequisite non-rust build dependencies
-* cmake configure `cmake . -B build`
-    * This downloads idl files. (TODO: optimize this.)
-* cmake build `cmake --build build`
-    * Generates fabric import libs for fabric dlls from fabric dlls installed with fabric runtime. 
-    * Build all rust examples.
+Now, proceed to the regular Ubuntu setup: [Getting Started - Ubuntu](#Getting-Started---Ubuntu)
 
-### Rust build
-You can use cargo to do rust only targets.
-* Compile or run rust example
-    * compile rust sdk `cargo build`
-    * run sample client executable `cargo run -p samples_client`. source code is in `crates\samples\client\src\main.rs`
-### Other targets
-* Build tcp echo service fabric singlton application (part of default build).
-    * `cmake --build build --target build_rust_sample_echomain`
-* Test the echoapp in local cluster (Windows)
-    * Add app to cluster `.\scripts\echomain_ctl.ps1 -Action Add`
-    * Run echo powershell to talk to the app `.\scripts\echomain_ctl.ps1 -Action Echo`
-    * Remove app from cluster `.\scripts\echomain_ctl.ps1 -Action Remove`
-* Build tcp echo stateful service fabric single partition application (part of default build).
-    * `cmake --build build --target build_rust_sample_echomain_stateful`    
+Edit the `/etc/wsl.conf` to the following and restart WSL to apply the changes:
+```sh
+# This removes windows path
+# [interop]
+# appendWindowsPath = false
+# This do not mount windows drive
+[automount]
+enabled = false
```

Comment:
Automount should be turn back to be true after sf installation.

Reply:
Addressed